### PR TITLE
Remove semantic markup

### PR DESCRIPTION
**Pull request by davidelanfranchi**
## Description
A small change to replace the semantic added markup with div tags.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by idiotWu** [comment]
Thanks!

Would you mind removing the dist files? Because they will be generated right before publishing.

File: dist/smooth-scrollbar.css [hunk 1]
```diff
@@ -1 +1 @@
...e;opacity:0;z-index:1;-webkit-transition:opacity .5s ease-out,background .5s ease-out;transition:opacity .5s ease-out,background .5s ease-out;background:none;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}[data-scrollbar] .scrollbar-track.show,[data-scrollbar] .scrollbar-track:hover,[scrollbar] .scrollbar-track.show,[scrollbar] .scrollbar-track:hover,scrollbar .scrollbar-track.show,scrollbar .scrollbar-track:hover{opacity:1}[data-scrollbar] .scrollbar-track:hover,[scrollbar] .scrollbar-track:hover,scrollbar .scrollbar-track:hover{background:hsla(0,0%,87%,.75)}[data-scrollbar] .scrollbar-track-x,[scrollbar] .scrollbar-track-x,scrollbar .scrollbar-track-x{bottom:0;left:0;width:100%;height:8px}[data-scrollbar] .scrollbar-track-y,[scrollbar] .scrollbar-track-y,scrollbar .scrollbar-track-y{top:0;right:0;width:8px;height:100%}[data-scrollbar] .scrollbar-thumb,[scrollbar] .scrollbar-thumb,scrollbar .scrollbar-thumb{position:absolute;top:0;left:0;width:8px;height:8px;background:rgba(0,0,0,.5);border-radius:4px}[data-scrollbar] .overscroll-glow,[scrollbar] .overscroll-glow,scrollbar .overscroll-glow{position:absolute;top:0;left:0;width:100%;height:100%}
```
Comment (idiotWu) [inline dist/smooth-scrollbar.css]: Oh, please, please don't delete them.

I would suggest you run `git reset --hard a8d8dd9`.

Comment (davidelanfranchi) [inline dist/smooth-scrollbar.css]: Sorry, never used actively git and I'm not sure I'll have time today to understand what's wrong :|.

Comment (idiotWu) [inline dist/smooth-scrollbar.css]: The dist files are **needed**, but it's **unnecessary** to update them in every commit. Instead I will run `npm run dist` right before publishing new versions 😆

**Comment by davidelanfranchi** [comment]
Thank you very much for the help @idiotWu! And sorry for the mess :)...

**Pull request merged**